Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt

```diff
@@ -197,11 +197,17 @@ internal class MonthView @JvmOverloads constructor(
       if (value != null) {
         field = value
         labelsViewModel.update(value.calendarLabels)
-        isRtl = value.isRtl
+        isRtl = layoutDirection == LAYOUT_DIRECTION_RTL
```

Comment:
Does this need to be set on line 109 and here?

Reply:
This is just to keep the existing behaviour as much intact as possible. We can remove it in the future. I prefer to do big code changes gradually, step by step.

Reply:
Makes sense 👍